src/subapps/dataExplorer/styles.less

```diff
@@ -12,6 +12,12 @@
     margin-bottom: 28px;
   }
 
+  .flex-container {
+    display: flex;
+    max-width: 90vw;
+    justify-content: space-between;
+  }
```

Comment:
can we have also `align-items: center` just so both sides of flex container not showed shifted since the toggle is bigger than the text in the left